src/org/zaproxy/zap/extension/callback/ExtensionCallback.java

```diff
@@ -202,6 +215,7 @@ public boolean onHttpRequestSend(HttpMessage msg) {
                                 .appendAsync(str + "\n");
                     }
                     LOGGER.info(str);
+                    callBackReceived(Constant.messages.getString("callback.handler.test.name"), msg.cloneAll());
```

Comment:
We could avoid cloning here? (The message is not modified by the view?)

Reply:
right. 
Should we then pass the not cloned httpMsg in line 234 as well? 
https://github.com/zaproxy/zaproxy/pull/3930/files/c93e31154200c379931cdb645528c4570cc4c515?diff=split#diff-a43a4e0e802023f741461882a6dd4a65R234

Reply:
Yeah, it could use `msg` there too.